src/main/java/mil/dds/anet/search/sqlite/SqliteAuthorizationGroupSearcher.java

```diff
@@ -1,91 +1,23 @@
-    final StringBuilder sql = new StringBuilder(
-        "/* SqliteAuthorizationGroupSearch */ SELECT * FROM \"authorizationGroups\"");
+  public SqliteAuthorizationGroupSearcher() {
+    super(new SqliteSearchQueryBuilder<AuthorizationGroup, AuthorizationGroupSearchQuery>(
+        "SqliteAuthorizationGroupSearch"));
+  }
 
+  @Override
+  protected void addTextQuery(AuthorizationGroupSearchQuery query) {
+    qb.addWhereClause(
+        "(\"authorizationGroupUuid\".name LIKE '%' || :text || '%' OR \"authorizationGroupUuid\".description LIKE '%' || :text || '%')");
     final String text = query.getText();
-    final boolean doFullTextSearch = (text != null && !text.trim().isEmpty());
-    if (doFullTextSearch) {
-      whereClauses.add("(name LIKE '%' || :text || '%' OR description LIKE '%' || :text || '%' )");
-      sqlArgs.put("text", Utils.getSqliteFullTextQuery(text));
-    }
-
-    if (query.getStatus() != null) {
-      whereClauses.add("status = :status");
-      sqlArgs.put("status", DaoUtils.getEnumId(query.getStatus()));
-    }
-
-    if (query.getPositionUuid() != null) {
-      // Search for authorization groups related to a given position
-      whereClauses
-          .add("uuid IN ( SELECT \"authorizationGroupUuid\" FROM \"authorizationGroupPositions\" "
-              + "WHERE \"positionUuid\" = :positionUuid) ");
-      sqlArgs.put("positionUuid", query.getPositionUuid());
-    }
-
-    final AnetBeanList<AuthorizationGroup> result = new AnetBeanList<AuthorizationGroup>(
-        query.getPageNum(), query.getPageSize(), new ArrayList<AuthorizationGroup>());
-
-    if (!whereClauses.isEmpty()) {
-      sql.append(" WHERE ");
-      sql.append(Joiner.on(" AND ").join(whereClauses));
-    }
-
-    // Sort Ordering
-    final List<String> orderByClauses = new LinkedList<>();
-    if (query.getSortBy() == null) {
-      query.setSortBy(AuthorizationGroupSearchSortBy.NAME);
-    }
-    if (query.getSortOrder() == null) {
-      query.setSortOrder(SortOrder.ASC);
-    }
-    switch (query.getSortBy()) {
-      case CREATED_AT:
-        orderByClauses.addAll(Utils.addOrderBy(query.getSortOrder(), null, "\"createdAt\""));
-        break;
-      case NAME:
-      default:
-        orderByClauses.addAll(Utils.addOrderBy(query.getSortOrder(), null, "name"));
-        break;
-    }
-    orderByClauses.addAll(Utils.addOrderBy(SortOrder.ASC, null, "uuid"));
-    sql.append(" ORDER BY ");
-    sql.append(Joiner.on(", ").join(orderByClauses));
-
-    sql.append(" LIMIT :limit OFFSET :offset");
-
-    final List<AuthorizationGroup> list = getDbHandle().createQuery(sql.toString()).bindMap(sqlArgs)
-        .bind("offset", query.getPageSize() * query.getPageNum()).bind("limit", query.getPageSize())
-        .map(new AuthorizationGroupMapper()).list();
-
-    result.setList(list);
-    // Sqlite cannot do true total counts, so this is a crutch.
-    result.setTotalCount(result.getList().size());
```

Comment:
I think we are missing the totalcount in Sqlite queries now.

Reply:
That is now done in the `AnetBeanList` constructor.

Reply:
SQLite support has now been dropped in favour of PostgreSQL, which can do counts the same way as MS SQL Server.